runtime/prelude.jakt

```diff
@@ -131,6 +131,10 @@ extern class File {
     public function exists(anon path: String) -> bool
 }
 
+import extern "IO/File.cpp" {
+    extern function mkdir(anon path: String) throws
+}
```

Comment:
This should import from the header file, not the cpp.

However, since this is the prelude, and the prelude already includes the IO/File.h header it should instead just be a free function

```suggestion
extern function mkdir(anon path: String) throws
```